packages/scripts/package.json

```diff
@@ -0,0 +1,38 @@
+{
+	"name": "@wordpress/scripts",
+	"version": "1.0.1-0",
+	"description": "Collection of JS scripts for WordPress development",
+	"author": "WordPress",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"scripts",
+		"reusable-scripts"
```

Comment:
Can we add `wordpress-scripts` please 🙏 

(It would fit in nicely listed here: https://github.com/reyronald/awesome-toolkits#other)

Reply:
Sure thing :)